### DISCUSSION TITLE
Java installing

### DISCUSSION THREAD

**Discussion by Leaf-Craft-Official** [blueedgetechno/win11React#466] [Q&A] [answered]
How can i install java?
If i start the .exe the file is running on my own pc!
Plsssssi need help!
![NO](https://user-images.githubusercontent.com/105841704/169711360-91319264-9df4-4fe1-99fa-51fd18e70c7b.gif)

**Comment by luni-moon** [comment, marked as answer]
You can’t.

On Sun, May 22, 2022 at 14:57, Leaf Craft Team ***@***.***>
wrote:

> How can i install java?
> If i start the .exe the file is running on my own pc!
> Plsssssi need help!
> [image: NO]
> <https://user-images.githubusercontent.com/105841704/169711360-91319264-9df4-4fe1-99fa-51fd18e70c7b.gif>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/blueedgetechno/win11React/discussions/466>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANI5RBNBJ7NKTJGZOBULVJ3VLJ7S7ANCNFSM5WTY3N4Q>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
-- 
Best Wishes,
Lukas Batema (Java, JavaScript, and Front End Web Developer; Moderator on
PedestriaMC Network)
